### DISCUSSION TITLE
Constant red light

### DISCUSSION THREAD

**Discussion by ThatsJustDaniel** [iRobotEducation/create3_docs#290] [Questions]
Hello, I bought my robot a few weeks ago and today it just stopped working while on its dock. The robot has a constant red light and does not respond to the power down button or standby mode button. Could someone help me on this issue please. I tried to access its main page via ip address, but most of the time it does not work. I was able to access it twice after multiple attempts, but it is also very slow.

**Comment by ThatsJustDaniel** [comment]
I took off the battery and put it back in and it works now. I would like to know the problem, but I don't really know what it was. Thanks btw if you are looking at this. Sorry for the inconvenience. I tried to erase this comment, but I couldn't find how.

**Reply by alsora** [reply]
Hi, a constant red light usually indicates that the Create 3 application is not running on the robot (i.e. it crashed and it's unable to start again).
You should still be able to access the Create 3 webserver and you can get the logs there https://iroboteducation.github.io/create3_docs/webserver/logs/.
Without the logs there isn't much I can say, besides standard troubleshooting.

**Comment by ThatsJustDaniel** [comment]
Hello, I took the battery off and put it back on and solved the issue. However, I would like to reset the robot to its factory settings and all memory, so I can start with a cleaner code. Do you know how to do a factory reset or wipe the memory ?  I already restarted the application on the web home page, but that did not erase the memory.

Thanks

Sent from my iPad

On Dec 29, 2022, at 10:56 AM, Alberto Soragna ***@***.***> wrote:

﻿

Hi, a constant red light usually indicates that the Create 3 application is not running on the robot (i.e. it crashed and it's unable to start again).
You should still be able to access the Create 3 webserver and you can get the logs there https://iroboteducation.github.io/create3_docs/webserver/logs/.
Without the logs there isn't much I can say, besides standard troubleshooting.

**Reply by alsora** [reply]
Hi @ThatsJustDaniel,
we recommend to open separate discussions if you have new questions that are not necessarily related to the original one.
I did it for you and added the answer on how to factory reset the robot here: https://github.com/iRobotEducation/create3_docs/discussions/291
